src/actions/grant.js

```diff
@@ -127,16 +138,19 @@ export function submitGrant() {
         dispatch(showValidationErrors({ form: errorMessages }));
         return false;
       }
+
+      const submissionId = data.createSubmission.id;
+      log('success!');
+      dispatch(submitGrantSuccess(submissionId));
+      history.push('/grant/success');
```

Comment:
Not sure if this is the right place for this. Feels cleaner than having it in the reducer. I love how elm does this with commands returned in the reducer. Much nicer than the redux pattern.